### ISSUE TITLE
Calendar print not satisfying anymore

### ISSUE THREAD

**Issue by computersalat** [Intermesh/groupoffice#1087]
Hi,

in the past I was able to print a month inluding the list of this month. I loved this feature very much.
Now when I switch view to month and select `print` -> `month` I get one calender page of this month. But the list is sadly missing.
When I try to print current view I am getting an error (see attachment).
When switch view to list I can only print a bunch 3 months. There is neither an option to select a specific month nor any filter based on a category to print.
Or did I miss something ?
![20231228_GO_print_current_view_error](https://github.com/Intermesh/groupoffice/assets/161880/b78ba854-e5a7-4448-866c-0a4bc636187f)

**Comment by computersalat** [comment]
ErrorException in /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php at line 129: Undefined property: GO\Calendar\Views\Pdf\CalendarPdf::$pageWidth
Debug info:
```
GO\Base\Data\JsonResponse::__set_state(array(
   'data' => 
  array (
    'success' => false,
    'exceptionCode' => 0,
    'exceptionClass' => 'ErrorException',
    'trace' => 
    array (
      0 => '#0 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php(129): go\\core\\ErrorHandler::errorHandler()',
      1 => '#1 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php(58): GO\\Calendar\\Views\\Pdf\\CalendarPdf->_processEvents()',
      2 => '#2 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/controller/EventController.php(1756): GO\\Calendar\\Views\\Pdf\\CalendarPdf->setParams()',
      3 => '#3 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/controller/EventController.php(1055): GO\\Calendar\\Controller\\EventController->_createPdf()',
      4 => '#4 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/controller/AbstractController.php(492): GO\\Calendar\\Controller\\EventController->actionStore()',
      5 => '#5 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/controller/AbstractController.php(390): GO\\Base\\Controller\\AbstractController->callActionMethod()',
      6 => '#6 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/Router.php(179): GO\\Base\\Controller\\AbstractController->run()',
      7 => '#7 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/index.php(127): GO\\Base\\Router->runController()',
      8 => '#8 {main}',
    ),
  ),
))

-----

EXCEPTION: ErrorException: Undefined property: GO\Calendar\Views\Pdf\CalendarPdf::$pageWidth in /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php:129
Stack trace:
#0 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php(129): go\core\ErrorHandler::errorHandler()
#1 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php(58): GO\Calendar\Views\Pdf\CalendarPdf->_processEvents()
#2 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/controller/EventController.php(1756): GO\Calendar\Views\Pdf\CalendarPdf->setParams()
#3 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/controller/EventController.php(1055): GO\Calendar\Controller\EventController->_createPdf()
#4 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/controller/AbstractController.php(492): GO\Calendar\Controller\EventController->actionStore()
#5 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/controller/AbstractController.php(390): GO\Base\Controller\AbstractController->callActionMethod()
#6 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/Router.php(179): GO\Base\Controller\AbstractController->run()
#7 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/index.php(127): GO\Base\Router->runController()
#8 {main}
ErrorException in /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php at line 129: Undefined property: GO\Calendar\Views\Pdf\CalendarPdf::$pageWidth
#0 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php(129): go\core\ErrorHandler::errorHandler()
#1 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/views/pdf/CalendarPdf.php(58): GO\Calendar\Views\Pdf\CalendarPdf->_processEvents()
#2 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/controller/EventController.php(1756): GO\Calendar\Views\Pdf\CalendarPdf->setParams()
#3 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/modules/calendar/controller/EventController.php(1055): GO\Calendar\Controller\EventController->_createPdf()
#4 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/controller/AbstractController.php(492): GO\Calendar\Controller\EventController->actionStore()
#5 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/controller/AbstractController.php(390): GO\Base\Controller\AbstractController->callActionMethod()
#6 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/go/base/Router.php(179): GO\Base\Controller\AbstractController->run()
#7 /srv/www/vhosts/go.wittmer.services/groupoffice-6.8.26/index.php(127): GO\Base\Router->runController()
#8 {main}
```

**Comment by derjoachim** [comment]
Fixed. To be released in the next 6.8 version.
